modMcf/src/org/aion/mcf/account/Keystore.java

```diff
@@ -231,7 +231,7 @@ public static ECKey getKey(String _address, String _password) {
         if (_address.startsWith(AION_PREFIX)) {
             List<File> files = getFiles();
             for (File file : files) {
-                if (file.getName().contains(_address)) {
+                if (file.getName().split("--")[2].equals(_address)) {
```

Comment:
we can't assume all the files in the keystore folder are the correct naming format. Need to add more check.

Reply:
Prefer a regex match check for hexadecimal string of 64 chars (or something similar)